packages/p2p/src/components/my-ads/my-ads.scss

```diff
@@ -26,19 +35,31 @@
             }
         }
         &-scrollbar {
-            position: absolute;
             height: calc(100% - 2.4rem - 92px);
+            position: absolute;
```

Comment:
```suggestion
            height: calc(100% - 2.4rem - 9.2rem);
            position: absolute;
```

Reply:
This is not something I've added but I've changed it.